swatch-contracts/src/main/java/com/redhat/swatch/contract/model/ContractMapper.java

```diff
@@ -154,6 +154,17 @@ default String getRhEntitlementSku(List<RhEntitlementV1> rhEntitlements) {
     }
   }
 
+  @Named("rhSubscriptionNumber")
+  default String getRhSubscriptionNumber(List<RhEntitlementV1> rhEntitlements) {
+    if (Objects.nonNull(rhEntitlements)
```

Comment:
Completely optional/stylistic...but you could simplify using a ternary operator if you wanted.
```
return Objects.nonNull(rhEntitlements) && !rhEntitlements.isEmpty() && Objects.nonNull(rhEntitlements.get(0)) ? rhEntitlements.get(0).getSubscriptionNumber() : null;
```